chrome/content/zotero/feedSettings.xul

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<?xml-stylesheet href="chrome://global/skin/" type="text/css"?>
+<?xml-stylesheet href="chrome://zotero/skin/zotero.css" type="text/css"?>
+<!DOCTYPE window [
+	<!ENTITY % zoteroDTD SYSTEM "chrome://zotero/locale/zotero.dtd" > %zoteroDTD;
+]>
+<dialog xmlns="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
+	title="&zotero.feedSettings.title;" buttons="cancel,accept"
+	buttonlabelaccept="&zotero.feedSettings.saveButton.label;"
+	ondialogaccept="Zotero_Feed_Settings.accept()"
+	ondialogcancel="Zotero_Feed_Settings.cancel()"
+	id="zotero-feed-settings"
+	onload="Zotero_Feed_Settings.init()">
+	
+	<script src="include.js"/>
+	<script src="feedSettings.js"/>
+	
+	<vbox>
+		<hbox align="center">
+			<label value="&zotero.feedSettings.url.label;" control="feed-url"/>
+			<textbox id="feed-url" flex="1" type="search" size="2"
+				oninput="Zotero_Feed_Settings.invalidateUrl()"
+				oncommand="Zotero_Feed_Settings.validateUrl()"
+				focused="true" newlines="replacewithspaces"
```

Comment:
As long as this is a URL field, might as well do `newlines="stripsurroundingwhitespace"`, which trims internal and external whitespace.